.ci/setup_env_sles.sh

```diff
@@ -0,0 +1,70 @@
+#!/bin/bash
+#
+# Copyright (c) 2019 Intel Corporation
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+set -e
+
+cidir=$(dirname "$0")
+source "/etc/os-release" || source "/usr/lib/os-release"
+source "${cidir}/lib.sh"
+
+echo "Add repo for perl-IPC-Run"
+perl_repo="https://download.opensuse.org/repositories/devel:languages:perl/SLE_12_SP3/devel:languages:perl.repo"
+sudo -E zypper addrepo ${perl_repo}
+sudo -E zypper refresh
+
+echo "Install perl-IPC-Run"
+sudo -E zypper -n install perl-IPC-Run
+
+echo "Add repo for moreutils"
+moreutils_repo="https://download.opensuse.org/repositories/utilities/SLE_12_SP3_Backports/utilities.repo"
+sudo -E zypper addrepo ${moreutils_repo}
+sudo -E zypper refresh
+
+echo "Install chronic"
+sudo -E zypper -n install moreutils
+
+echo "Install curl"
+chronic sudo -E zypper -n install curl
+
+echo "Install git"
+chronic sudo -E zypper -n install git
+
+echo "Install kata containers dependencies"
+chronic sudo -E zypper -n install libtool automake autoconf bc libpixman-1-0-devel coreutils
+
+echo "Install qemu dependencies"
+chronic sudo -E zypper -n install libcap-devel libattr1 libcap-ng-devel librbd-devel
+
+echo "Install kernel dependencies"
+chronic sudo -E zypper -n install libelf-devel flex
+
+echo "Install bison binary"
+chronic sudo -E zypper -n install bison
+
+echo "Install libudev-dev"
+chronic sudo -E zypper -n install libudev-devel
+
+echo "Install Build Tools"
+chronic sudo -E zypper -n install python zlib-devel pattern Basis-Devel
+
+echo "Install tools for metrics tests"
+chronic sudo -E zypper -n install  jq
+
+if [ "$(arch)" == "x86_64" ]; then
+	echo "Install Kata Containers OBS repository"
+	obs_url="${KATA_OBS_REPO_BASE}/SLE_12_SP3/"
```

Comment:
@marcov is it possible to change the URL for the obs packages to get match with the info of `/etc/os-release`
```
NAME="SLES"
VERSION="12-SP3"
VERSION_ID="12.3"
PRETTY_NAME="SUSE Linux Enterprise Server 12 SP3"
ID="sles"
ANSI_COLOR="0;32"
CPE_NAME="cpe:/o:suse:sles:12:sp3"
```

Reply:
Hi @GabyCT, that's something possible, but then we need to make sure we don't break something else like [installation instructions](https://github.com/kata-containers/documentation/blob/master/install/sles-installation-guide.md). 

For the moment, could you build the URL like this? :
`obs_url="${KATA_OBS_REPO_BASE}/SLE_${VERSION//-/_}"`

Reply:
@marcov changes applied

Reply:
thank you @GabyCT